tests/utils/axis-utils-tests.js

```diff
@@ -32,7 +32,7 @@ test('axis-utils #getTicksTotalFromSize', t => {
 
 test('axis-utils #getTickValues', t => {
   const scale = scaleLinear().domain([0, 1]).range(['red', 'blue']);
-  t.deepEqual(getTickValues(scale, 10, false),
+  t.deepEqual(getTickValues(scale, 10, false).map(d => Math.round(d * 1000) / 1000),
```

Comment:
I know @apercu introduced changes recently that caused this test to have these values, rounding them invalidates the value of this test